spec/javascripts/Policies/components/CustomPolicy.spec.jsx

```diff
@@ -22,7 +20,19 @@ describe('CustomPolicy', () => {
     expect(wrapper.find('header').hasClass('CustomPolicy-header')).toBe(true)
     expect(wrapper.find('h2').hasClass('CustomPolicy-title')).toBe(true)
     expect(wrapper.find('a').hasClass('CustomPolicy-cancel')).toBe(true)
-    expect(wrapper.find('Form').prop('initialPolicy')).toBe(POLICY_TEMPLATE)
+    expect(wrapper.find('CustomPolicyEditor').prop('initialPolicy')).toBe(POLICY_TEMPLATE)
+  })
+})
+
+describe('CustomPolicyEditor', () => {
+  it('should render itself correctly', () => {
+    const wrapper = shallow(<CustomPolicyEditor initialPolicy={POLICY_TEMPLATE} />)
+    expect(wrapper.find('SchemaEditor').exists()).toBe(true)
+    expect(wrapper.find('Form').exists()).toBe(true)
+    expect(wrapper.find('.PolicyConfiguration-name').text()).toBe('Name of the policy')
+    expect(wrapper.find('.PolicyConfiguration-version').text()).toBe('0.0.1')
```

Comment:
Same for the following lines you can use the imported `POLICY_TEMPLATE`

Reply:
Same as above ;)

Reply:
That way we are testing the `POLICY_TEMPLATE` for free

Reply:
OK, agree